internal/docker/docker_config.go

```diff
@@ -98,6 +98,11 @@ func CreateDockerCompose(s *types.Stack) *DockerComposeConfig {
 				},
 				Logging: StandardLogOptions,
 			}
+
+			if s.PrometheusEnabled {
+				compose.Services["firefly_core_"+member.ID].Ports = append(compose.Services["firefly_core_"+member.ID].Ports, fmt.Sprintf("%d:%d", member.ExposedFireflyMetricsPort, member.ExposedFireflyMetricsPort))
+			}
```

Comment:
I'm actually not sure that this is required. Does the metrics port really need to be exposed? Or does it just need to be available within the docker compose network? If there's a use case for it being port forwarded to the host, I'm more than happy to leave it. Otherwise, I think we should try to limit the number of ports the stack needs to reduce the likelihood of conflicts with other services running on the host machine.

Reply:
Good point, I think _most_ use cases wouldn't benefit from it being exposed. If you're working with the Prometheus metrics, then you'll be able to use Prometheus itself to query them.

I could see it being handy if you need to debug the metrics being served from FireFly, but that can be done using a `docker exec ... curl` kinda thing so I agree that exposing them isn't necessary at this time. Will remove.